app/src/main/java/com/example/lastfm/Post.java

```diff
@@ -0,0 +1,39 @@
+package com.example.lastfm;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Post {
+    @SerializedName("name")
+    @Expose
```

Comment:
What does the annotation mean?

Reply:
That class have no use right now. Deleted